dockerfiles/wolfi-py3.12-slim/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:experimental
+FROM cgr.dev/chainguard/wolfi-base:latest
+
+USER root
+RUN apk update && \
+    apk add bash python-3.12 py3.12-pip && \
```

Comment:
python-3.12 is not needed
```
docker run -it cgr.dev/chainguard/wolfi-base

470ca44c3cf1:/# python
/bin/sh: python: not found
470ca44c3cf1:/# python3.12
/bin/sh: python3.12: not found
470ca44c3cf1:/# apk add py3.12-pip
fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
(1/16) Installing libbz2-1 (1.0.8-r8)
(2/16) Installing libexpat1 (2.6.2-r1)
(3/16) Installing libffi (3.4.6-r3)
(4/16) Installing gdbm (1.24-r0)
(5/16) Installing xz (5.6.2-r0)
(6/16) Installing libgcc (13.3.0-r2)
(7/16) Installing libstdc++ (13.3.0-r2)
(8/16) Installing mpdecimal (4.0.0-r2)
(9/16) Installing ncurses-terminfo-base (6.5_p20240629-r0)
(10/16) Installing ncurses (6.5_p20240629-r0)
(11/16) Installing readline (8.2-r7)
(12/16) Installing sqlite-libs (3.45.1-r1)
(13/16) Installing python-3.12-base (3.12.4-r1)
(14/16) Installing py3.12-setuptools (72.1.0-r0)
(15/16) Installing py3.12-pip-base (24.2-r0)
(16/16) Installing py3.12-pip (24.2-r0)
OK: 86 MiB in 30 packages
470ca44c3cf1:/# python3.12
Python 3.12.4 (tags/v3.12.4-dirty:8e8a4ba, Jun 26 2024, 12:07:40) [GCC 13.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 
```

Reply:
I'll keep ti make explicit that we install python, not only pip